crates/rpc/src/handlers/validator.rs

```diff
@@ -30,6 +32,8 @@ pub struct ValidatorData {
     validator: Validator,
 }
 
+pub const MAX_REQUEST_LENGTH: usize = 1000;
```

Comment:
We already have a const `MAX_VALIDATOR_COUNT` could we consolidate these and only use 1?